openbb_terminal/cryptocurrency/onchain/bitquery_model.py

```diff
@@ -816,4 +816,7 @@ def get_spread_for_crypto_pair(
     return df
 
 
-POSSIBLE_CRYPTOS = list(get_erc20_tokens()["symbol"].unique())
```

Comment:
yeah, i also thought about this. i didn't like how it was implemented as a constant but it's actually loading stuff from disk

Reply:
Yes, it is - my issue with it was that this file gets "loaded" before the loggers are set up, and since `get_erc20_tokens` gets called and will try to log something, might lead to an unexpected behavior.